index.js

```diff
@@ -1,88 +1,119 @@
 // ***** Question 1 *****
+let counter = 1 
+counter = 2
 // *** Uncomment the lines below to test
-// console.log("%cQuestion 1", "color: red") 
+console.log("%cQuestion 1", "color: red") 
 
-// console.log(counter) 
-// // => 2
-// console.log("%c----------", "color: red") 
+console.log(counter) 
+// => 2
+console.log("%c----------", "color: red") 
 
 
 // ***** Question 2 *****
+const name = "jonathan"
```

Comment:
Nice! The key here was using `const` because it doesn't allow us to reassign our variable